documentation/docs/guides/tool-permissions.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Managing Tool Permissions
+sidebar_position: 4
+sidebar_label: Tool Permissions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Tool permissions provide fine-grained control over how Goose uses different tools within extensions. This guide will help you understand and configure these permissions effectively.
+
+## Understanding Tools and Extensions
+
+Before diving into permissions, let's clarify the key components:
+
+- **Extensions** are packages that add functionality to Goose (like Developer, Google Drive, etc.)
+- **Tools** are specific functions within each extension that Goose can use
+
```

Comment:
maybe excessive but i see a lot of people confused with the difference in our #goose-help channel